tests/test_lab_process_list.py

```diff
@@ -44,6 +44,16 @@ def test_process_list(cli_runner: CliRunner):
         ).isoformat(),
         "status": ILAB_PROCESS_STATUS.RUNNING,
     }
+    process_registry["63866b92-799a-42a7-af02-d0b68fddf19d"] = {
+        "pid": 26373,
+        "children_pids": [555, 666, 777],
+        "type": "Generation",
+        "log_file": "/Users/charliedoern/.local/share/instructlab/logs/generation/generation-63866b92-799a-42a7-af02-d0b68fddf19d.log",
+        "start_time": datetime.datetime.strptime(
+            start_time_str, "%Y-%m-%d %H:%M:%S"
+        ).isoformat(),
+        "status": ILAB_PROCESS_STATUS.DONE,
+    }
```

Comment:
(Note to myself, no change required) This code should be re-written to use ProcessRegistry class to manage the state.